ex7-structCond.js

```diff
@@ -0,0 +1,21 @@
+// - Exercice 7
+// - Ecrire un algorithme qui prend 3 nombres en entrée, stockés
+// dans 3 variables
+// - Vérifier si les nombres sont donnés dans l’ordre croissant
+// - Amélioration : remettez-les dans l’ordre
+
+let prompt = require("prompt-sync")();
+let a = parseFloat(prompt("Saisir le premier nombre :"));
+let b = parseFloat(prompt("Saisir le deuxième nombre :"));
+let c = parseFloat(prompt("Saisir le troisième nombre :"));
+let tab = [a, b, c];
+const CheckOrderNumbers = (a, b, c) => {
+  if (a < b && b < c) {
+    console.log("Les nombres sont dans l'ordre croissant");
+  } else {
+    console.log("Les nombres ne sont pas dans l'ordre croissant");
+  }
+};
```

Comment:
sinon tu peux utliser les fonction de tableau de es6 pour vérifier l'ordre par exemple 

```suggestion
const checkOrderNumbers = (a, b, c) => {
  const numbersInTab = [a,b,c]
  const isOrdered = [a,b,c].sort().join(' ') === [a,b, c].joind(' ')
  console.log(isOrdered ? "Les nombres sont dans l'ordre croissant" : "Les nombres ne sont pas dans l'ordre croissant")
};

checkOrderNumbers(1, 2, 3)
```